README.md

```diff
@@ -291,6 +307,19 @@ Check out the [Examples](examples/) folder to see different use cases.
 
   ![](assets/identical-keys-error-screenshot-2018-04-30.png)
 
+### i18n-json/sorted-keys
+
+- automatic ascending sort of all keys in the translation file.
+- default severity: error | 2
```

Comment:
just to keep it consistent, should we use `.` here too? or remove from the rest of the list items too?

Reply:
ah, yup, thanks!